application/features/behat.yml

```diff
@@ -23,6 +25,6 @@ default:
             contexts: [ Sil\Idp\IdSync\Behat\Context\WebhookContext ]
         workday_integration_features:
             filters:
-                tags: '@allUsers'
+                tags: '~@specificUser && ~@usersChangedSince && ~@canUpdateLastSynced'
```

Comment:
These tags mean that (for the Workday integration)...
- Don't run the tests for a specific user
- Don't run the tests for users-changed-since
- Don't run the tests for updating when users were last synced

Those are skipped because the Workday ID Store adapter doesn't support those features.